src/gui/plugins/visualize_contacts/VisualizeContacts.qml

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls 2.1
+import QtQuick.Dialogs 1.0
+import QtQuick.Layouts 1.3
+import "qrc:/qml"
+
+GridLayout {
+  columns: 6
+  columnSpacing: 10
+  Layout.minimumWidth: 200
+  Layout.minimumHeight: 200
+  anchors.leftMargin: 10
+  anchors.rightMargin: 10
+
+  // Left spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 15
+    Layout.fillWidth: true
+  }
+
+  CheckBox {
+    Layout.alignment: Qt.AlignHCenter
+    id: visualizeContacts
+    Layout.columnSpan: 4
+    text: qsTr("Show/Hide Contacts")
```

Comment:
I would suggest simply saying `Show Contacts` as opposed to `Show/Hide`. Right now checking the box is show, unchecking is hide. But before I did anything, it was confusing, "what would happen if I check it, would it show or hide?"

Reply:
I think the user would expect them to disappear when it's unchecked. So deleting seems more reasonable.